api/backup.go

```diff
@@ -76,6 +76,20 @@ func (e BackupStorageBackend) String() string {
 	return "UNKNOWN"
 }
 
+// BinlogInfo is the binlog coordination for MySQL.
+type BinlogInfo struct {
+	FileName string `json:"fileName"`
+	Position int64  `json:"position"`
+}
+
+// BackupPayload is encoded in JSON and stored in the backup table, representing PITR related info.
+type BackupPayload struct {
+	BinlogInfo BinlogInfo `json:"binlogInfo"`
```

Comment:
We need to comment when the binlog info is recorded during the backup process. Because we use this info to locate the backup, thus the timing is important.

Reply:
Done.

Reply:
It means that when we recover to the backup, the next valid binlog replay position is recorded in the `BinlogInfo`.
Do you have any suggestions to make it more clear? @tianzhou @d-bytebase

Reply:
>> It means that when we recover to the backup, the next valid binlog replay position is recorded in the

this seems to contradict the comment `BinlogInfo is recorded when taking the backup`

Reply:
Added comments referring to the design doc.
Also added an explanatory section in the design doc on why choosing such a seems-sophisticated backup process.